.github/workflows/cd-create-version-tag.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: cd-create-version-tag
+
+on: #  yamllint disable-line rule:truthy
+  push:
+    branches: [main]
+  pull_request:
```

Comment:
Should this be running when pull requests are open with target branch of main?

Reply:
This is interesting, because the `on` is `push:branches[main]` and `pull_request:branches:[main]` it'll only trigger when the pull request is merged into main. It essentially `ANDs` them together.

Reply:
Oh interesting! Learning new things about Github Actions every day haha